spec/widgets/auto-style.spec.js

```diff
@@ -7,11 +7,14 @@ describe('auto-style', function () {
     var layer = vis.map.layers.first();
     layer.restoreCartoCSS = jasmine.createSpy('restore');
     layer.set('initialStyle', '#layer {  marker-line-width: 0.5;  marker-line-color: #fcfafa;  marker-line-opacity: 1;  marker-width: 6.076923076923077;  marker-fill: #e49115;  marker-fill-opacity: 0.9;  marker-allow-overlap: true;}');
-    this.dataviewModel = vis.dataviews.createCategoryModel(layer, {
-      column: 'col'
+    var source = vis.analysis.findNodeById('a0');
```

Comment:
`findNodeById` is likely to be removed [here](https://github.com/CartoDB/cartodb.js/issues/1790), the new api encourages you to create the source from scratch:

`var source = vis.analyse(jsonSource)`

Reply:
I'm sure that means a HUGE refactor since most of tests here in `deep-insights` rely on a module call `spec-helper` that uses `createVis` to set a common scenario for all the tests.

If `findNodeById`is going to be replaced I think that refactor should be made on https://github.com/CartoDB/cartodb.js/issues/1790 integration tickets, not here.